### PR TITLE
Kubernetes wrapper

### LINKED ISSUE #203: Kubernetes Wrapper
As ZMON now runs within Kubernetes clusters, we need some easy-to-use Kubernetes wrapper to be used in ZMON check definitions. Example:

```python
# return number of ready pods for "myapp"
def check():
   pods = kubernetes().pods(application="myapp")
   return len([pod for pod in pods if pod['ready']])
```

The concrete API should be drafted/discussed before implementing.

### PR DESCRIPTION
**Pull request by mohabusama**
Resolve #203

### CODE REVIEW

File: zmon_worker_monitor/builtins/plugins/kubernetes.py [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+        query = pykube.PersistentVolume.objects(self.__client).filter(**filter_kwargs)
+
+        # PersistentVolume does not belong to a "namespace". Calling with **default** will fail with 404.
+        pvs = query.all()
+
+        return [vc.obj for vc in pvs if phase is None or vc.obj['status'].get('phase') == phase]
+
+    def metrics(self):
+        """
+        Return cluster wide metrics in prometheus format.
```
Comment (mikkeloscar) [inline zmon_worker_monitor/builtins/plugins/kubernetes.py]: I think I may have caused some confusion, this is not cluster wide metrics, but only metrics for the apiserver.

Comment (mohabusama) [inline zmon_worker_monitor/builtins/plugins/kubernetes.py]: Ah ok, will update it.

**Comment by mikkeloscar** [comment]
lgtm

File: zmon_worker_monitor/builtins/plugins/kubernetes.py [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+        :param phase: Pod status phase. Valid values are: Pending, Running, Failed, Succeeded or Unknown.
+        :type phase: str
+
+        :param ready: Pod ready status. If None then all pods are returned.
+        :type ready: bool
+
+        :param **kwargs: Pod labelSelector filters. Example: application__in=['app-1', 'app-2'], version='v0.1'
+                         Supported filter syntax:
+                            - <label>
+                            - <label>__in
```
Comment (Jan-M) [inline zmon_worker_monitor/builtins/plugins/kubernetes.py]: This looks odd but think labelselector does not look much different

**Comment by Jan-M** [comment]
:+1:

**Comment by lmineiro** [comment]
👍

**Comment by mohabusama** [comment]
👍

**Pull request merged**